internal/acceptance/openstack/compute/v2/migrate_test.go

```diff
@@ -26,6 +27,16 @@ func TestMigrate(t *testing.T) {
 
 	err = servers.Migrate(context.TODO(), client, server.ID).ExtractErr()
 	th.AssertNoErr(t, err)
+
+	allPages, err := migrations.List(client, migrations.ListOpts{
+		InstanceID: &server.ID,
+	}).AllPages(context.TODO())
+	th.AssertNoErr(t, err)
+
+	allMigrations, err := migrations.ExtractMigrations(allPages)
+	th.AssertNoErr(t, err)
+
+	th.AssertIntGreaterOrEqual(t, len(allMigrations), 1)
```

Comment:
You [found your issue](https://github.com/gophercloud/gophercloud/pull/3244#issuecomment-2515131802) already :)

We'll need to [add another compute](https://docs.openstack.org/devstack/latest/guides/multinode-lab.html) for the nova job to test migrations. @stephenfin do you have an idea if that's easily feasible?

@Patzifist in case you want to give it a shot, the job is configured in https://github.com/gophercloud/gophercloud/blob/master/.github/workflows/functional-compute.yaml.

Reply:
I suggest disabling the test and merging the request